index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+    
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<meta name="viewport" content="width=device-width,height=device-height,initial-scale=1.0"/>
+<title>Steven Universe - Adventure!</title>
+<!-- CSS -->
+<link rel="stylesheet" type="text/css" href="css/main.css"/>
+<link rel="stylesheet" type="text/css" href="css/responsive.css"/>
+<link rel="stylesheet" type="text/css" href="css/grid.css"/>
+</head>
+
+<body style="overflow:hidden;"> <!-- prevents vertical scroll -->
+
+<div class="grid">
+  <div class="row-1">*</div>
+  <div class="row-2">*</div>
+  <div class="row-3">*</div>
+  <div class="row-4">*</div>
+  <div class="row-5">
+    <h1><img src="images/Steven_Universe_logo.svg.png"/></h1>

```

Comment:
I was hoping this would be not an image and created using a font. That way we could do some cool animation